theano/tensor/nnet/__init__.py

```diff
@@ -103,6 +103,11 @@ def conv2d(input, filters, input_shape=None, filter_shape=None,
         Factor by which to subsample (stride) the input.
         Also called dilation elsewhere.
 
+    unshared: bool
+        If true, then unshared or 'locally connected' convolution will be
+        performed. A different kernel will be used for each region of the
+        input.
+
```

Comment:
I think it would be useful to change the documentation for `filters` as well, to explain how you should pass the unshared weights.